scripts/auto_back_up.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+# Copyright (C) 2019 Zilliqa
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import os
+import argparse
+import time
+import subprocess
+import json
+import socket
+import tarfile
+import xml.etree.cElementTree as xtree
+
+TAG_NUM_FINAL_BLOCK_PER_POW = "NUM_FINAL_BLOCK_PER_POW"
+TESTNET_NAME= "TEST_NET_NAME"
+BUCKET_NAME='BUCKET_NAME'
+AWS_PERSISTENCE_LOCATION= "s3://"+BUCKET_NAME+"/persistence/"+TESTNET_NAME
```

Comment:
can we convert all these into command-line options? This would provide a robust interface for users and avoid on-the-fly patching of the scripts.

Reply:
I agree to this to extend. It might work well enough for scripts like auto_back_up.py and uploadIncrDB.py which runs in our controlled testnet network and we can modify the invocation of these script everywhere with required parameters somehow.

however, for downloadIncrDB.py which is occassionally called by zilliqa process  instead of launch.sh/launch_docker.sh we need process to be aware of testnet specific parameters either through `constants.xml` or having `zilliqa` process itself take these as command line arguments and make then available to script like downloadIncrDB.py at runtime.
We need more work on this.

Reply:
Thanks for the response. Let's perhaps plan this in the future.